patches/minecraft/net/minecraft/entity/EntityLivingBase.java.patch

```diff
@@ -321,7 +321,15 @@
  
                      if (!itemstack.func_190926_b())
                      {
-@@ -2575,6 +2609,40 @@
+@@ -2377,6 +2411,7 @@
+             {
+                 flag = false;
+             }
++            flag = net.minecraftforge.common.ForgeHooks.canElytraFly(this, itemstack);
+         }
```

Comment:
What is your solution for ticking/damaging custom elytra items when its used? Force modders to have their own entity tick handlers?

Reply:
If needed, yes. It's not a very elegant solution, but I don't see a better alternative since attributes have many possible usages and combinations. Most implementations won't need a dedicated tick handler since items, armors, and potions have their own update methods.

Of course, if this is a sticking point, I'm open to suggestions on how to better handle this.